spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteManifestsAction.java

```diff
@@ -84,6 +89,7 @@
 import org.junit.jupiter.api.TestTemplate;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.io.TempDir;
+import scala.Tuple2;
```

Comment:
I would avoid using Scala classes here, we have a Pair utility class you can use in it's place

Reply:
Great, thanks!